app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    for individual in people:
+        person = Person(individual["name"], individual["age"])
+        person_list.append(person)
+
+    for individual in people:
+        if "wife" in individual.keys() and \
```

Comment:
please, don't use `\ `, instead you can create a shorter variable name (person for ex) and skip keys() method (`if "wife" in individual` is the same)

Reply:
Used tips and rewrote